.github/workflows/pythonpackage.yml

```diff
@@ -60,7 +60,7 @@ jobs:
             python-version: 3.8
             jpeg-deps: pillow-jpegls
             gdcm-dep: gdcm
-            pylibjpeg-deps: none, lj+oj
+            pylibjpeg-dep: normal
```

Comment:
Just a convention, but I put names like 'gdcm' for the values, because that is what shows up on the build summary lines on the PR front page -- just makes it a little easier to see which tests are being run in each.

Reply:
Fair enough.  I don't see that for some reason, maybe I'm not looking in the same place.